meta-mender-toradex-nxp/recipes-bsp/u-boot/files/colibri-imx7/0001-colibri-imx7-Adapt-u-boot-environment-settings-for-M.patch

```diff
@@ -0,0 +1,34 @@
+From b39a6a374cd6ebb9c9bc1ca17fc7f59364500f5a Mon Sep 17 00:00:00 2001
+From: Drew Moseley <drew.moseley@northern.tech>
+Date: Fri, 6 Dec 2019 15:07:25 +0000
+Subject: [PATCH] colibri-imx7: Adapt u-boot environment settings for Mender on
+ sumo.
```

Comment:
I have been running imx7 on sumo for a while now. Not sure why these changes are needed?

Reply:
Which branch of the Toradex layers are you using?

Reply:
I have been using the versions that are in the current manifest,

https://github.com/mendersoftware/meta-mender-community/blob/sumo/meta-mender-toradex-nxp/scripts/manifest-toradex.xml

Reply:
OK.  I looked in the repos themselves and didn't see a sumo branch so I was using rocko-next (I think).  I'll take another pass through with that manifest.

Reply:
Oh.  I just realized that was the meta-mender-community manifest and not one from Toradex. How did you identify those SHAs?

Reply:
Are you sure have used sumo and not rocko? I tried to do a clean checkout and build and get the following:

``
$ source setup-environment toradex-nxp
Sorry, it does not seem that *toradex-nxp* is a valid target

Supported targets are:
atmel
beaglebone
intel
nxp
qemu
raspberrypi
renesas
rockchip
sunxi
tegra
up
variscite
``

Reply:
> Are you sure have used sumo and not rocko?

Reply:
OK.  I still have to change the bootenv size to get it to build.  Are you seeing that?

Reply:
Nope, build passes just fine. What specific error are you getting?

Reply:
I think I know why we are each confused here. I'm building colibri-imx7 which has failures and you are building colibri-imx7-emmc which does not.

Reply:
> I think I know why we are each confused here. I'm building colibri-imx7 which has failures and you are building colibri-imx7-emmc which does not.

True